projects/aca-shared/rules/src/app.rules.spec.ts

```diff
@@ -1360,6 +1360,18 @@ describe('app.evaluators', () => {
     });
   });
 
+  describe('canPrintFile', () => {
+    it('should return false for media files', () => {
+      context.selection.file = { entry: { content: { mimeType: 'video/ogg' } } } as any;
```

Comment:
please use "as NodeEntry" instead of "as any"

Reply:
Updated